### ISSUE TITLE
Icon: Acceptatiecriteria opstellen

### ISSUE THREAD

**Issue by Yolijn** [nl-design-system/candidate#934]
## TLDR

We willen graag zorgen dat de Candidate component onderhoudbaar, flexibel en robuust is én toegankelijk kan worden ingezet. Daarom is het belangrijk dat we voordat we beginnen met het maken van het component goed nadenken over alle acceptatiecriteria. Zowel op technisch vlak als voor toegankelijkheid. Hierin is een hoop overlap, dus deze kunnen in samenwerking met een developer en een toegankelijkheidsspecialist worden opgepakt. 

## Oplossingsrichting

De acceptatiecriteria kunnen in eerste instantie toegevoegd worden aan dit issue, zodat ze daarna kunnen worden opgesplitst in acceptatiecriteria die interessant zijn voor op nldesignsystem.nl, stories in de test storybook én eventueel stories in de evil storybook voor acceptatiecriteria waarvan de veelgemaakte fouten bekend zijn.

## Actiepunten

- [ ] ...

## Acceptatiecriteria

- [x] Acceptatiecriteria vanuit toegankelijkheid nl-design-system/kernteam#1490 zijn opgeschreven in dit issue
- [x] Acceptatiecriteria vanuit development zijn opgeschreven in dit issue
- [x] De acceptatiecriteria zijn afgestemd met de Design System Lead
- [ ] De acceptatiecriteria zijn als FYI gedeeld met de community in het #nl-design-system-developer kanaal in Slack
```
💡 We zijn bezig om een Candidate versie van de Icon component beschikbaar te maken. Om te zorgen dat deze aan al onze eisen en wensen voldoet stellen we voordat we beginnen al onze acceptatiecriteria op. 

Benieuwd welke dat zijn? Je kunt ze hier [in een GitHub issue](https://github.com/nl-design-system/candidate/issues/934) vinden. En mocht je iets missen, dan maak je ons heel gelukkig als je dat hier 🧵 laat weten.
```
- [x] De acceptatiecriteria die passen bij de WCAG 2.1 AA criteria zijn beschikbaar gemaakt op nldesignsystem.nl
- [ ] De acceptatiecriteria die relevant zijn voor het toevoegen aan de test storybook zijn toegevoegd aan het issue [test stories toevoegen aan de candidate test storybook](https://github.com/nl-design-system/candidate/issues/933)
